src/components/expression/expression.tsx

```diff
@@ -35,10 +36,20 @@ const colorToClassNameMap = {
   danger: 'euiExpression--danger',
 };
 
+const textWrapToClassNameMap = {
+  'break-word': 'euiExpression-breakWord',
+  truncate: 'euiExpression-truncate',
+};
+
 export const COLORS = keysOf(colorToClassNameMap);
 
 export type ExpressionColor = keyof typeof colorToClassNameMap;
 
+const displayToClassNameMap = {
+  inline: 'euiExpression-inline',
+  columns: 'euiExpression-columns',
```

Comment:
Same comments as for the `textWrap` prop apply here.

```suggestion
  inline: null,
  columns: 'euiExpression--columns',
```